src/main/java/pl/szczesniak/dominik/whattowatch/movies/domain/MoviesToWatchService.java

```diff
@@ -28,10 +30,31 @@ public void removeMovieFromList(final MovieId movieId, final UserId userId) {
 		repository.removeMovie(movieId, userId);
 	}
 
-	public List<Movie> getList(final UserId userId) {
+	public List<Movie> getMoviesToWatch(final UserId userId) {
 		return repository.findAll(userId);
 	}
 
+	public List<WatchedMovie> getWatchedMovies(final UserId userId) {
+		return watchedRepository.findAllBy(userId);
+	}
+
+	public void moveMovieToWatchedList(final MovieId movieId, final UserId userId) {
+		userCheck(userId);
+		final WatchedMovie watchedMovie = new WatchedMovie(
+				movieId,
+				repository.findBy(movieId, userId).orElseThrow(() -> new MovieDoesNotExistException("Movie doesn't match userId: " + userId)).getTitle(),
```

Comment:
moglibyśmy to wyciągnać do prywatnej metody getById. Będzie tutaj krócej, ukryjemy sobie szczegół jakim jest to, że filmu może nie być, a dodatkowo podejrzewam, że tworzenie WatchedMovie zmieści nam się wtedy w jednej linii